jsonnet/jsonnetfile.json

```diff
@@ -4,11 +4,11 @@
     {
       "source": {
         "git": {
-          "remote": "https://github.com/coreos/etcd",
+          "remote": "https://github.com/etcd-io/etcd",
           "subdir": "Documentation/etcd-mixin"
         }
       },
-      "version": "master"
+      "version": "release-3.4"
```

Comment:
FYI: there is no more branch master, as mentioned on prom-operator slack channel, etcd renamed it main now. This is the version we use in 4.6 so pinned to that.